packages/devextreme/js/ui/tab_panel.d.ts

```diff
@@ -18,6 +18,7 @@ import {
 
 import {
     Position,
+    TabsIconPosition,
```

Comment:
Add reexport:

```ts
import {
    TabsIconPosition,
} from '../common';
```

Reply:
It's the rule for all components - all public types should be reexported

Reply:
Was updated here [e20fbd7cff57e643bde9fe631f671be9dbb154a4].